### PR TITLE
Added StringToListConverter Documentation

### PR DESCRIPTION
**Pull request by GUOLDEV**
## Fixes # 

NA

## What changes to the docs does this PR provide?

Added StringToListConverter Documentation

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] For new pages, used the [provided template](https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/master/docs/.template.md).
- [x] For new features, added an entry in the [Table of Contents](https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/master/docs/TOC.yml).
- [x] Ran against a spell and grammar checker.
- [x] Contains **NO** breaking changes.

### CODE REVIEW

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[cfc63b5](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/cfc63b5ef80b0eb49217f53597f174326218d90f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C31%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202103310514341003-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hwDu%2BPeI5VmX5OC%2BsWVtcInMdjZu6K0WnKEovKpW%2Fiw%3D&st=2021-03-31T05%3A10%3A19Z&se=2021-05-01T05%3A15%3A19Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[8af3f14](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/8af3f142186e9e4d8c6507d986018be034e94942)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C31%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202103310519485391-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=930UwI16XvTR7rOxJ9MK98%2B57PGYiuRqan71xRQLhl8%3D&st=2021-03-31T05%3A15%3A29Z&se=2021-05-01T05%3A20%3A29Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[213e436](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/213e4362ae4988db439cb43d728f0be762e5298f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C31%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202103310530019896-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ZAs6BpOoS3f7Y4Wz00zOK9z8biEBrTg3MWz1sswIEl0%3D&st=2021-03-31T05%3A25%3A39Z&se=2021-05-01T05%3A30%3A39Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[dd47336](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/dd4733610adf26d22169dcf56fe5523394d67358)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C31%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202103310533195118-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Ye5oZY5eDUJNd%2FEXX9za2Dxj00DNwoPgd6%2Bj4B9c%2BRk%3D&st=2021-03-31T05%3A28%3A56Z&se=2021-05-01T05%3A33%3A56Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[1267239](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/1267239e8ea6a398c8e1fe1f218ea1968620533f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) | [Details](#user-content-9376f0b8c44cda0cbced95a371fc2c5e-19ae48c4)
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

<a id="9376f0b8c44cda0cbced95a371fc2c5e-19ae48c4"></a>
### [docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md)
 - **Line 1, Column 1**: **[Warning-[author-missing](https://review.docs.microsoft.com/help/contribute/validation-ref/alt-text-missing?branch=main)]** `````Missing required attribute: 'author'. Add the current author's GitHub ID.`````
 - **Line 1, Column 1**: **[Warning-[title-missing](https://review.docs.microsoft.com/help/contribute/validation-ref/title-missing?branch=main)]** `````Missing required attribute: 'title'. Add a title string to show in search engine results.`````
 - **Line 1, Column 1**: **[Warning-[ms-author-missing](https://review.docs.microsoft.com/help/contribute/validation-ref/ms-author-missing?branch=main)]** `````Missing required attribute: 'ms.author'. Add the current author's Microsoft alias.`````
 - **Line 3, Column 18**: **[Warning-yaml-header-syntax-error]** `````While parsing a block mapping, did not find expected key.`````
 - **Line 1, Column 1**: **[Suggestion-[description-missing](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C31%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202103310538125241-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=p8SC8BOCsZQscqdS9m0BqagTN9Ilo75Xr2RYifOIZUU%3D&st=2021-03-31T05%3A33%3A55Z&se=2021-05-01T05%3A38%3A55Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[4b32387](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/4b32387c67178682073c3469359497a4e9fee8d3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C31%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202103310540220433-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=BOb4lQLTLhzZJDGgBQ8dkpcLxohq1XAXXIG0a4XY8%2Fk%3D&st=2021-03-31T05%3A36%3A02Z&se=2021-05-01T05%3A41%3A02Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jfversluis** [comment]
Note that this has not been merged yet in XCT

File: docs/converters/stringtolistconverter.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+## Properties
+
+<!-- Explain all properties in a table format -->
+
+| Property | Type | Description |
+| -- | -- | -- |
+| Separator | string | The string that delimits the substrings in this string. |
+| Separators | string[] | The strings that delimits the substrings in this string. |
```
Comment (pictos) [inline docs/converters/stringtolistconverter.md]: The `Type` shouldn't be `IList<string>` ?

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[8c9d28c](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/8c9d28c2699fc6de5795cdc2688f8409e85ccefc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C31%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202103312038158398-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=grTlxuwTLw5CgncjoLb%2BsXIxUQWdFQN5jYhDLU5dONM%3D&st=2021-03-31T20%3A34%3A04Z&se=2021-05-01T20%3A39%3A04Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/converters/stringtolistconverter.md (hunk 1 above)
Comment (GUOLDEV) [reply docs/converters/stringtolistconverter.md]: @pictos Thanks for the review, I have fixed it, but the `<string>` part is not displayed in the [file](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md). Do you know how to fix it?

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[b1ea4bc](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/b1ea4bc3855f58f3fe1c29efb8113fa914689757)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C3%5C31%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202103312049403251-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=3ICdagos9w4HfTRXvH1FH3rwuRNxRCqVFIz6ZVp95jY%3D&st=2021-03-31T20%3A45%3A29Z&se=2021-05-01T20%3A50%3A29Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/converters/stringtolistconverter.md (hunk 1 above)
Comment (GUOLDEV) [reply docs/converters/stringtolistconverter.md]: @pictos never mind, I have found out how to display special characters.

**Review by pictos — APPROVED** [review]

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[c300718](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/c30071855ef20890ba55bebb7e5e9fed71f7be0f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C21%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202105211026426153-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=oUeFwSuaax1NUdnOLb6e3BmodIx7XsULTXiU%2FTWgbMw%3D&st=2021-05-21T10%3A22%3A15Z&se=2021-06-21T10%3A27%3A15Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by jfversluis — APPROVED** [review]

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[c633b2b](https://github.com/GUOLDEV/xamarin-communitytoolkit/commits/c633b2b681f6efb35ee78115da1d13569ed9190b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |
[docs/converters/stringtolistconverter.md](https://github.com/GUOLDEV/xamarin-communitytoolkit/blob/stringtolistconverter-doc/docs/converters/stringtolistconverter.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/converters/stringtolistconverter?branch=pr-en-us-109) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C5%5C21%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202105211310318150-109%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=a4xtKXb%2FvnsudA5L%2FLZWd2fmC8w4rUy4zf1k%2Bb17UFo%3D&st=2021-05-21T13%3A06%3A06Z&se=2021-06-21T13%3A11%3A06Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Pull request merged**